src/App.jsx

```diff
@@ -1,7 +1,17 @@
+import { Routes, Route } from 'react-router-dom';
 import List from './components/ListChampions';
+import Navbar from './components/Navbar';
 
 function App() {
-  return <List />;
+  return (
+    <div>
+      <Navbar />
+      <Routes>
+        <Route path='/' element={<div>Hello World!</div>} />
```

Comment:
Les strings doivent être entre `{''}`
```suggestion
        <Route path='/' element={<div>{'Hello World!'}</div>} />
```